Bear359 patch 1 (#14783)
hashicorp/consul: website/content/docs/integrate/partnerships.mdx

> -> **Network Infrastructure Automation (NIA)***: These integrations leverage Consul's service catalog to seamlessly integrate with Consul-Terraform-Sync (CTS) to automate changes in network infrastructure via a publisher-subscriber method. More details can be found [here](/docs/integrate/nia-integration).

Reviewer: ```suggestion
-> **Network Infrastructure Automation (NIA)***: These integrations leverage Consul's service catalog to seamlessly integrate with Consul-Terraform-Sync (CTS) to automate changes in network infrastructure via a publisher-subscriber method. Refer to the [NIA documentation](/docs/integrate/nia-integration) for details.
```

> **HCP Consul**: HCP Consul is secure by default and offers an out-of-the-box service mesh solution to streamline operations without the hassle of managing Consul servers. Sign up for HCP Consul is free and available [here](https://cloud.hashicorp.com/products/consul).

Reviewer: ```suggestion
**HCP Consul**: HCP Consul is secure by default and offers an out-of-the-box service mesh solution to streamline operations without the hassle of managing Consul servers. [Sign up for a free HCP Consul account](https://cloud.hashicorp.com/products/consul).
```
It's good practice to link keywords and phrases, rather than "here", "this link", etc.

#### Data Plane:

**Application Performance Monitoring (APM)**

- [Consul Telemetry Documentation](/docs/agent/telemetry)
- [Monitoring Consul with Datadog APM](https://www.datadoghq.com/blog/consul-datadog/)
- [Monitor HCP Consul with New Relic Instant Observability](https://github.com/newrelic-experimental/hashicorp-quickstart-annex/blob/main/hcp-consul/README.md)

> - [HCP Consul & CloudFabrix AIOps Integration](https://bot-docs.cloudfabrix.io/Bots/consul/?h=consul)

Reviewer: ```suggestion
- [HCP Consul and CloudFabrix AIOps Integration](https://bot-docs.cloudfabrix.io/Bots/consul/?h=consul)
```
Avoid using ampersands

- [Consul and SnappyFlow Full Stack Observability](https://docs.snappyflow.io/docs/integrations/hcp_consul)

**Network Performance Monitoring (NPM)**

- [Datadog NPM now supports Consul networking](https://www.datadoghq.com/blog/monitor-consul-with-datadog-npm/)

**OpenTelemetry Integrations**

- [Splunk SignalFX OpenTelemetry integration with Consul](https://docs.splunk.com/Observability/gdi/consul/consul.html)
- [Ship HashiCorp Consul metrics with OpenTelemetry to Logz.io](https://docs.logz.io/shipping/prometheus-sources/consul.html)
- [Ingest Consul metrics through OpenTelemetry into Lightstep Observability](https://docs.lightstep.com/docs/ingest-metrics-consul)

> **Logging & Alerting**

Reviewer: ```suggestion
**Logging and Alerts**
```

- [Consul Integration with iLert](https://docs.ilert.com/integrations/consul)
- [Consul Integration with PagerDuty](https://www.pagerduty.com/docs/guides/consul-integration-guide/)
- [Monitor Consul with Zabbix](https://www.zabbix.com/integrations/hashicorp_consul#consul)

> **API Gateway & Ingress Controller**

Reviewer: ```suggestion
**API Gateway and Ingress Controller**
```

  - [Automate VMware Advanced Load Balancers (Avi) with Consul NIA](https://www.hashicorp.com/integrations/_vmware/consul)

**Application Delivery Controllers \(ADC\)**

- [Automate A10 ADC with Consul NIA](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-a10-adc?utm_source=docs)
- [Automate Citrix ADC with Consul NIA](https://www.hashicorp.com/integrations/citrix-adc/consul)

**Domain Name Service (DNS) Automation**

- [Automate DNSimple public facing DNS records with Consul NIA](https://registry.terraform.io/modules/dnsimple/cts/dnsimple/latest)
- [Automate NS1 managed DNS with Consul NIA](https://github.com/ns1-terraform/terraform-ns1-record-sync-nia)

**No-Code/Low-Code**

- [Automate Consul Deployments with Sophos Factory Pipelines](https://community.sophos.com/sophos-factory/f/recommended-reads/136639/deploy-hashicorp-consul-from-sophos-factory)

### 3. Develop and Test

The only knowledge necessary to write a plugin is basic command-line skills and knowledge of the [Go programming language](http://www.golang.org). Use the plugin interface to develop your integration. All integrations should contain unit and acceptance testing.

> **HCP Consul**: The process to configure a testing instance of HCP consul [is very simple](https://learn.hashicorp.com/tutorials/cloud/consul-introduction?utm_source=docs). HCP has been designed as a HashiCorp managed service so configuration is minimal as only Consul client agents need to be installed. Furthermore, HashiCorp provides all new users an initial credit which should last approximately 2 months using a [development cluster](https://cloud.hashicorp.com/products/consul/pricing). When deployed with AWS or Azure free tier services, there should be no cost beyond the time spent by the designated tester.

Reviewer: ```suggestion
**HCP Consul**:  As a managed service, minimal configuration is required to deploy HCP Consul server clusters. You only need to install Consul client agents. Furthermore, HashiCorp provides all new users an initial credit, which provides approximately two months worth of [development cluster](https://cloud.hashicorp.com/products/consul/pricing) access. When deployed with AWS or Azure free tier services, there should be no cost beyond the time spent by the designated tester. Refer to the [Deploy HCP Consul tutorial](https://learn.hashicorp.com/tutorials/cloud/consul-introduction?utm_source=docs) for details on getting started.
```
Just a few things w/r/t our style guide: 
- avoid simplifying language
- use active voice
- spell out numbers 0-9

> Please note that HCP Consul is currently only deployed on AWS and Microsoft Azure so the partner’s application should be able to be deployed or run in AWS or Azure. 

Reviewer: ```suggestion
HCP Consul is currently only deployed on AWS and Microsoft Azure, so your application can be deployed to or run in AWS or Azure. 
```
Right? We're speaking to the partner, so we can just say "your application", correct?

#### HCP Consul Resource Links:

- [Getting Started with HCP Consul](https://learn.hashicorp.com/tutorials/cloud/consul-introduction?utm_source=docs)
- [HCP Consul End-to-End Deployment](https://learn.hashicorp.com/tutorials/cloud/consul-end-to-end-overview?in=consul/cloud-deploy-automation)
- [Deploy HCP Consul with EKS using Terraform](https://learn.hashicorp.com/tutorials/cloud/consul-end-to-end-eks?in=consul/cloud-deploy-automation)

> - [HCP Consul Deployment Automation](https://learn.hashicorp.com/collections/consul/cloud-deploy-automation)

Reviewer: ```suggestion
- [HCP Consul Deployment Automation](https://learn.hashicorp.com/collections/consul/cloud-deploy-automation)
- [HCP Consul documentation]( https://cloud.hashicorp.com/docs/consul/usage)
```
How about a link to the actual documentation?
